index

```diff
@@ -1,3 +1,6 @@
+6e457c6185ac293b5c533ee3284c8fb289a7721f7711cc5980bf632020452439 irma-demo/.idea/modules.xml
```

Comment:
did you push the intellij folder by accident?

Reply:
Apparently `irma scheme sign` includes those files in the index. That was not the intention. I'll remove them.

Reply:
Now it should be better.